helpers/session.js

```diff
@@ -55,7 +55,21 @@ module.exports = class Session extends Helper {
     return new Promise((resolve, reject) => {
       this._getSession()
         .then(sessionData => {
-          resolve(sessionData[`${SESSION_KEY_PREFIX}-${sessionKey}`] || {});
+          let key;
+          if (!sessionKey) {
+            const keys = Object.keys(sessionData);
+            if (keys.length === 3) {
```

Comment:
um ok, so how do you know that there is a sessionKey if the key has a length of 3? Is it just assumed that if there are three then they must be sessionKey, cookie and exists?